addons/website/tests/test_ui.py

```diff
@@ -221,3 +221,6 @@ def test_12_edit_translated_page_redirect(self):
         lang = self.env['res.lang']._activate_lang('nl_NL')
         self.env['website'].browse(1).write({'language_ids': [(4, lang.id, 0)]})
         self.start_tour("/nl/contactus", 'edit_translated_page_redirect', login='admin')
+
+    def test_11_carousel_snippet_content_removal(self):
```

Comment:
@tdecaluwe When you push force branches in stable with a test, don't forget to check the test number. Here should be renamed to test_13, I'll do it myself now :+1: